pkg/kubectl/cmd/taint.go

```diff
@@ -321,6 +321,11 @@ func (o TaintOptions) RunTaint() error {
 		return err
 	}
 
+	smPatchVersion, err := cmdutil.GetServerSupportedSMPatchVersionFromFactory(o.f)
```

Comment:
You cannot error and exit here - servers that don't export /version (for security reasons) are now broken.  All uses of this method must ignore errors.  This is a blocker for 1.5 @saad-ali